lang/da/registration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+return [
+  /*|--------------------------------------------------------------------------
+    | Custom labels
+    |--------------------------------------------------------------------------
+    | These labels are used in views created by us (when locale is da)
+    |
+    */
+
+    'public_index_tittle_welcome' => 'Hold tilmeldings oversigt',
```

Comment:
Not something that HAS to be corrected now, but 'title' only has one 't' in the middle.

Reply:
Fixed